_data/changelogs/template-documentation.yml

```
@@ -0,0 +1,16 @@
+title: Documentation page
+type: template
+items:
+  - date: NNNN-NN-NN
+    summary: Updated the order of the side navigation markup to match the visual order. 
+    summaryAdditional:  
+    isBreaking: false
```

@mejiaj, @mahoneycm 
Do we consider this to be a breaking change? I lean towards yes but wanting to check in since it is a bit borderline.
```suggestion
    isBreaking: true
```

---

Noting that I agree this is probably a good idea since it affects the template markup in the component code section. Resolved in [b26cbf2](https://github.com/uswds/uswds-site/pull/2495/commits/b26cbf28fad0b4af03975aedcedef76bb282c892) !